assets/custom-css/utilities/_sgds-example.scss

```
@@ -13,6 +12,13 @@
         grid-auto-flow: column;
         grid-auto-columns: 1fr;
     }
+    @include media-breakpoint-down(sm) {
+        padding: 0.25rem;
+    }
+    @include media-breakpoint-up(sm) {
```

no need to include media breakpoint up here, the media-breakpoint-down will kick in when the viewport is sm and below
```suggestion
    padding: 2rem
```